sso-kit-starter/src/main/java/com/vaadin/sso/starter/hilla/endpoint/AuthEndpoint.java

```diff
@@ -0,0 +1,202 @@
+/*-
+ * Copyright (C) 2022-2023 Vaadin Ltd
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ *
+ * See <https://vaadin.com/commercial-license-and-service-terms> for the full
+ * license.
+ */
+package com.vaadin.sso.starter.hilla.endpoint;
```

Comment:
I'd prefer something more Hilla-centric, like `dev.hilla.sso`. This package could even live in a separate artifact: even Hilla users could choose not to use it, for example those who need to stick to old Hilla generator.

But, at least, it should be a sibling of `com.vaadin.sso.starter`, to allow for selective choice in `scanBasePackages`. This is needed to be able to whitelist the endpoind separately. See https://github.com/vaadin/sso-kit-hilla/blob/43117cfdcb826447b8b8acac4c2dbbcd41c28584/sso-kit-hilla-demo/src/main/java/com/example/application/Application.java#L12-L13